_rules/audio-is-live-213x3x.md

```diff
@@ -19,13 +19,13 @@ acknowledgments:
 This rule applies to every `audio` element that is:
 
 - playing; or,
-- has a "play button" that is [visible][] and [included in the accessibility tree][].
-
-**Note:** A play button is an interactive element that when activated, plays the audio.
+- has a [play button][] that is [visible][] and [included in the accessibility tree][].
 
 ## Expectation 
 
-The target element's [media resource](https://html.spec.whatwg.org/multipage/media.html#media-resource) is being broadcasted in real time without being first recorded and stored on the origin server.
+For each test target one of the following is true:
+- it is not possible to [seek][] the end of the target element's [media resource][]; or 
```

Comment:
I like that 👍 
OTOH, I'm not sure it's possible to "not be possible to seek". The algorithm seems to always return (possibly after doing nothing). Would that rather be part of the second bullet (and so the first can be removed)?

Reply:
I believe you are correct. I've updated the expectation accordingly.